### ISSUE TITLE
Wine Weird not converted correctly

### ISSUE THREAD

**Issue by ariakas2008** [surged20/foundryvtt-update-image-macro#4]
For some reason, the Wine Weird from ToA didn't convert correctly, both in token and in artwork.
I used the local path (being 5et/img/ToA/Wine%20Weird.png).
I ended fixing them manually.

**Comment by surged20** [comment]
> For some reason, the Wine Weird from ToA didn't convert correctly, both in token and in artwork.
> I used the local path (being 5et/img/ToA/Wine%20Weird.png).
> I ended fixing them manually.

I'll see if I can reproduce this on the current version. Sounds like everything else was updated properly?

**Comment by surged20** [comment]
Was this the actor in the sidebar or a placed token on a scene? If the latter, was the token linked or unlinked?

**Comment by ariakas2008** [comment]
unlinked placed token on a scene. Everything else worked fine. Maybe the %20 replacing the space in the name of the png file? dunno.

**Comment by surged20** [comment]
Ok, part of it makes sense. The macro currently does not update the synthetic actor image data within a unlinked token. It's probably time to address that in the next update.

However, I did verify that the token image itself updated with this specific case. That should always work so I can explain it. the %20 is handled without a problem. Here's what the unlinked token image and synthetic actor data look like prior to update:

```
canvas.tokens.controlled[0].data.img
"https://5e.tools/img/ToA/Wine%20Weird.png"
canvas.tokens.controlled[0].actor.data.img
"https://5e.tools/img/ToA/Wine%20Weird.png"
```

and after update to local image path:

```
canvas.tokens.controlled[0].data.img
"modules/plutonium/img/ToA/Wine%20Weird.png"
canvas.tokens.controlled[0].actor.data.img
"https://5e.tools/img/ToA/Wine%20Weird.png"
```
where the latter synthetic actor not getting updated is expected at this time.

**Comment by surged20** [comment]
Closing as this should be addressed in #5.
